### PR TITLE
chore(test): split the access log of apisix and test server

### PR DESCRIPTION
**Pull request by soulbird**
### Description

1. Record the logs of apisix and test server separately for better observation
2. Use the defined log format

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

**Comment by spacewander** [comment]
Please make the CI pass, thanks!

File: t/plugin/ext-plugin/http-req-call.t [hunk 1]
```diff
@@ -514,8 +514,8 @@ GET /hello
             ext.go({rewrite_bad_path = true})
         }
     }
---- access_log
-GET /plugin_proxy_rewrite_args%3Fa=2
+--- error_log
+undefined path in test server, uri: /plugin_proxy_rewrite_args?a=2
```
Comment (spacewander) [inline t/plugin/ext-plugin/http-req-call.t:518]: IMHO, we can use `request_uri` so we know it is a "bad" path.

Comment (soulbird) [reply t/plugin/ext-plugin/http-req-call.t:518]: The access log of the test server has been placed in another file. --- access_log It seems that the access_log file path cannot be specified.

Comment (spacewander) [reply t/plugin/ext-plugin/http-req-call.t:518]: I mean using it in the error log. Like:
```
ngx.log(ngx.WARN, "undefined path in test server, uri: ", ngx.var.request_uri)
```

**Review by spacewander — APPROVED** [review]

**Review by monkeyDluffy6017 — APPROVED** [review]

**Pull request merged**